tests/ops/test_qubit_ops.py

```diff
@@ -923,6 +923,19 @@ def test_iswap_eigenval(self):
         res = op.eigvals
         assert np.allclose(res, exp)
 
+    def test_swap_decomposition(self):
+        "Tests the swap operator produces the correct output"
+        opr = qml.SWAP(wires=[0, 1])
+        decomp = opr.decomposition([0, 1])
+        mat = []
+        for op in reversed(decomp):
+            if isinstance(op, qml.CNOT) and op.wires.tolist() == [0, 1]:
+                mat.append(np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]]))
```

Comment:
We could use the `CNOT` matrix import from `gate_data` at the top:
```python
from gate_data import (
    (...)
    CNOT,
```

```suggestion
                mat.append(CNOT)
```